doc/source/index.rst

```diff
@@ -8,14 +8,33 @@ Pyorbital
 
 Pyorbital is a python package to compute orbital parameters for satellites from
 TLE files as well as astronomical parameters of interest for satellite remote sensing.
-Currently pyorbital only supports low earth orbit satellites.
+Currently Pyorbital only supports low earth orbit satellites.
+
 
 Installation
 ------------
-Pyorbital comes with a file platforms.txt that maps satellite name to NORAD identifier.
-This file needs to be copied to the appropriate satpy etc directory ($PPP_CONFIG_DIR).
-It is wise to check it contains your satellites of interest. The NORAD identifier can
-be found as the first number of each line in the Two-Line Elements (eg. from celestrak).
+Pyorbital comes with a file *platforms.txt* that maps a satellite name to the NORAD identifier.
```

Comment:
This section is called "Installation", but the entire section is about customizing an installation that, as it says below, should not be needed by most users as the builtin platforms file should be sufficient.

Could this section "Installation" be renamed to something like "Add missing platform information" and instead have an installation section that talks about `pip install pyorbital` or `conda install -c conda-forge pyorbital`?

Reply:
Good suggestion! Fixed, but tried keeping the installation part very short and simple. Okay?

Reply:
I rewrote the instructions. I think it matches what I've written for other pytroll packages. I think it is important to mention the exact pip and conda commands for the usual use case. The installing from source commands are also important for the two cases of "install the dev version" cases for user support/bug fixes and the "you want to contribute, install in editable mode". I say that because I've heard users request these instructions over and over and over again.

Reply:
Looks fine, thx!
Realize that RTD does not build for PRs in this repo. Have forgot what I did for other projects to get this working.
Can we merge this then?

Reply:
Looks like RTD builds have been failing for quite some time. Let me look into it.

Reply:
I've enabled pull requests now too.

Reply:
Ah it's been building on Python 2.7.

Reply:
Looks good now. One of us can merge when the tests pass again.